Updated 2 websites with api request

degun-osint opened the pull request.

degun-osint committed:
Updated Tunefind

Previous method did not work anymore.
Added API request to get user and kept previous uri as uri_pretty

degun-osint committed:
Updated Quitter.pl

added API resquest and kept previous URI as uri_pretty
updated known username as previous username were deleted.

WebBreacher commented on wmn-data.json:
```diff
@@ -4839,12 +4839,13 @@
-        "e_string" : "@quitter.pl",
-        "m_string" : "Nie znaleziono",
+        "e_string" : "avatar_static",
+        "m_string" : "error",
```
Can you change this to `Can't find user` or `\"error\": \"` to make it more unique than just the string `error`?

degun-osint replied:
I added `\"error\":` because the next message can change based on browser langage.
I got `{"error":"Compte non trouvé"}` for example.

degun-osint committed:
Update quitter.pl

degun-osint committed:
Update quitter.pl

degun-osint committed:
Updated Wikidot

Changed m_code to 200 and updated e_string and m_string

degun-osint committed:
Deleted Yelp.com

Now use full JS website with no difference between existing or non existing user.
Same http code for both, and need registration in order to see user profile.

degun-osint committed:
Update isMyGirl

API check was using a parking domain.
Now responding with HTTP 200 to every request.
Updated cat to NSFW as it is a porn site.

degun-osint committed:
Update Vero

now m_code = 200
updated m_string

degun-osint committed:
Update Weibo

m_code = 200
changed m_string and e_string

degun-osint committed:
Delete Curiouscat

website closed in october 2024

degun-osint commented:
It's just some adjustments and deletion proposal for non working website.
At least I made a different commit for each one :D 

(next time I'll wait)

degun-osint committed:
Update pinterest

code 200 everytime, updated strings

degun-osint committed:
Update behance.net

seems that behance is always responding with a 400 when request is made without proper browser

degun-osint committed:
Merge pull request #1 from WebBreacher/main

merge commit

degun-osint committed:
added booknode

degun-osint committed:
Merge pull request #2 from WebBreacher/main

merge with origin

WebBreacher commented:
@degun-osint - I've been working on your and the other massive PRs for over a month. Many of your submissions are awesome and easily approved. But, since they are included in this PR with lots of other changes, I cannot merge just them.

I'd be really happy if, in the future, you'd make multiple, smaller PRs of maybe 5 site changes so that I can process pieces and get them into the project faster.

WebBreacher commented on wmn-data.json:
```diff
@@ -530,8 +530,8 @@
         "e_code" : 200,
-        "e_string" : "og:site_name",
-        "m_string" : "<title>Oops!",
+        "e_string" : "class=\"ProfileCard-userFullName-ule",
```
I'm not seeing this in the response. My response is all javascript that then makes graphql calls to pull the data. Can you check this please? We may need to remove this site.

degun-osint replied:
Yes. It worked one time, but now I doesn't work anymore.
I can't find any difference between existing or non existing account.

WebBreacher commented on wmn-data.json:
```diff
@@ -3018,14 +3017,13 @@
         "name" : "isMyGirl",
-        "uri_check" : "https://api.fxcservices.com/pub/user/{account}",
-        "uri_pretty" : "https://ismygirl.com/{account}",
+        "uri_check" : "https://ismygirl.com/models/{account}",
```
this should probably have a `uri_check` of `https://ismygirl.com/api/model/getModel?slug={account}`

degun-osint replied:
updated, with ne strings and known users

degun-osint committed:
deleted behance

degun-osint committed:
updated ismygirl

added uri_check
updates strings and known users

degun-osint commented on wmn-data.json:
```diff
@@ -3018,14 +3017,13 @@
         "name" : "isMyGirl",
-        "uri_check" : "https://api.fxcservices.com/pub/user/{account}",
-        "uri_pretty" : "https://ismygirl.com/{account}",
+        "uri_check" : "https://ismygirl.com/models/{account}",
```
see other comment

degun-osint commented on wmn-data.json:
```diff
@@ -3018,14 +3017,13 @@
-        "e_string" : "display_name",
-        "m_code" : 404,
-        "m_string" : "User does not exist",
+        "e_string" : "<a title=\"About me\">",
```
see other comment

degun-osint commented on wmn-data.json:
```diff
@@ -3018,14 +3017,13 @@
-        "m_string" : "User does not exist",
+        "e_string" : "<a title=\"About me\">",
+        "m_code" : 200,
+        "m_string" : "No model was found",
```
see other comment

degun-osint committed:
updated ismygirl

degun-osint commented on wmn-data.json:
```diff
@@ -4518,9 +4516,9 @@
-        "e_string" : " - Profile | Pinterest",
-        "m_string" : "Whoops! We couldn't find that page",
-        "m_code" : 404,
+        "e_string" : "id=\"business-profile-header",
```
It can be different based on langage.
But tools should be in english so I switched back to original string.

degun-osint commented on wmn-data.json:
```diff
@@ -4518,9 +4516,9 @@
-        "m_string" : "Whoops! We couldn't find that page",
-        "m_code" : 404,
+        "e_string" : "id=\"business-profile-header",
+        "m_string" : "pinterest.com/?show_error=true",
```
response is just a 301 code, with no particular strings.

degun-osint commented on wmn-data.json:
```diff
@@ -6097,10 +6096,11 @@
        },
        {
         "name" : "Tunefind",
```
done

degun-osint committed:
Updated upon comment by webbreacher

WebBreacher committed:
Merge branch 'main' into main
